### PR TITLE
Update prepare_deployment_branch.yaml

### PR DESCRIPTION
**Pull request by etanb**
Change time of cutting branch from 3:00pm PST to 7:00pm PST

### CODE REVIEW

File: .github/workflows/prepare_deployment_branch.yaml [hunk 1]
```diff
@@ -6,7 +6,7 @@ on:
     # At 23:00 on Monday and Wednesday (https://crontab.guru/#0_16_*_*_1,3)
```
Comment (snesm) [inline .github/workflows/prepare_deployment_branch.yaml:6]: Please update comments related to the new time/day

**Review by snesm — CHANGES_REQUESTED** [review]

File: .github/workflows/prepare_deployment_branch.yaml (hunk 1 above)
Comment (etanb) [reply .github/workflows/prepare_deployment_branch.yaml]: Updated!

**Review by jeremy-page — APPROVED** [review]
So this will run at 03:00 on Tuesday and Thursday, is that your intent?

**Comment by etanb** [comment]
@jeremy-page unless i'm mistaken, this task will now run at 3:00 AM (UTC) on every Tuesday and Thursday which translates to 7:00 PM PST on every Monday and Wednesday.

**Comment by etanb** [comment]
@snesm we good to merge?

**Review by snesm — APPROVED** [review]
@etanb looks good!

**Pull request merged**